test/end2end/test_messagebus_restart.py

```
@@ -0,0 +1,205 @@
+    @classmethod
+    def tearDownClass(cls):
+        # tear everything down while the bus is still up, then close the
+        # clients before stopping the bus: emitting on a disconnected client
+        # blocks its emitter worker until a reconnect, hanging the runner
+        time.sleep(2)
+        if cls.core_bus.connected_event.is_set():
+            try:
+                cls.skill_loader.unload()
+                cls.service.shutdown()
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)